src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateObject = { ...state };
+  const result = [];
+
+  for (const key of actions) {
```

Comment:
It is a clean code but as per checklist 
2.[CODE STYLE]: Use switch statement if you have limited amount of conditions.